Makefile

```diff
@@ -36,7 +36,7 @@ IMAGE_TAG_BASE ?= open-cluster-management.io/search-v2-operator
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
 # Image URL to use all building/pushing image targets
-IMG ?= quay.io/stolostron/search-v2-operator:latest
+IMG ?= quay.io/stolostron/search-v2-operator:2.7.0-dc0fc8c59239b6a0a3f440b7752101523d87f597
```

Comment:
why 2.7 image?

Reply:
I used the value that's currently in the csv at bundle/manifests to reduce the changes to the csv.

These get overwritten later by installer.
Maybe we could change it to something like `quay.io/stolostron/search-v2-operator:tag-placeholder`

Reply:
ok, thanks.